src/Bicep.MSBuild.E2eTests/.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
```

Comment:
Worth just adding this to the top-level `.editorconfig` for `[*.{ts,tsx,js}]` rather than creating new ones?

Reply:
Yeah good pt.

Reply:
Done.